bgpd/bgp_bmp.c

```diff
@@ -570,6 +641,14 @@ static void bmp_send_all(struct bmp_bgp *bmpbgp, struct stream *s)
 	stream_free(s);
 }
 
+static void bmp_send_all_safe(struct bmp_bgp *bmpbgp, struct stream *s)
+{
+	if (!s || !bmpbgp)
```

Comment:
Shouldn't this be &&? Because if bmpbgp is NULL, then bmp_send_all() would dereference it...?

Reply:
changed to 

if (!bmpbgp)
    stream_free(s);